typescript/web/src/connectors/undo-store/effects/update-label.ts

```diff
@@ -31,7 +26,17 @@ const updateLabelMutation = gql`
   }
 `;
 
-const getLabelQuery = gql`
+const IMAGE_DIMENSIONS_QUERY = gql`
+  query imageDimensions($id: ID!) {
+    image(where: { id: $id }) {
+      id
+      width
+      height
+    }
+  }
+`;
+
+const GET_LABEL_QUERY = gql`
   query getLabelAndGeometry($id: ID!) {
```

Comment:
So you do that here, but @bbenoist undos it in #834, who is right here, and why?